Failed: LocalBootstrapTest

See https://travis-ci.org/akka/akka-management/jobs/501511298

```
[info] LocalBootstrapTest:
[info] Cluster bootstrap with health checks
[info] - should not be ready initially *** FAILED *** (11 seconds, 57 milliseconds)
[info]   The code passed to eventually never returned normally. Attempted 6 times over 11.034055459000001 seconds. Last failure message: The future returned an exception of type: akka.stream.StreamTcpException, with message: The connection closed with error: Connection reset by peer.. (LocalBootstrapTest.scala:91)
[info]   org.scalatest.exceptions.TestFailedDueToTimeoutException:
[info]   at org.scalatest.concurrent.Eventually.tryTryAgain$1(Eventually.scala:432)
[info]   at org.scalatest.concurrent.Eventually.eventually(Eventually.scala:439)
[info]   at org.scalatest.concurrent.Eventually.eventually$(Eventually.scala:391)
[info]   at akka.management.LocalBootstrapTest.eventually(LocalBootstrapTest.scala:57)
[info]   at akka.management.LocalBootstrapTest.$anonfun$new$5(LocalBootstrapTest.scala:91)
[info]   at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[info]   at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
[info]   at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
[info]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:20)
[info]   at org.scalatest.WordSpecLike$$anon$1.apply(WordSpecLike.scala:1075)
```

[localbootstrap.logs.gz](https://github.com/akka/akka-management/files/2931065/localbootstrap.logs.gz)

Also:

```
[ERROR] [03/04/2019 15:04:19.567] [local-cluster-akka.actor.default-dispatcher-6] [akka://local-cluster@localhost:45129/system/IO-TCP/selectors/$a/1] Bind failed for TCP channel on endpoint [localhost/127.0.0.1:45427]
java.net.BindException: Address already in use
```

--- a/integration-test/local/src/test/scala/akka/management/LocalBootstrapTest.scala
+++ b/integration-test/local/src/test/scala/akka/management/LocalBootstrapTest.scala
@@ -17,7 +17,7 @@ import org.scalatest.time.{ Seconds, Span }
 import org.scalatest.{ Matchers, WordSpec }
 
 object LocalBootstrapTest {
-  val managementPorts = SocketUtil.temporaryServerAddresses(3, "127.0.0.1" ).map(_.getPort)
+  val managementPorts = SocketUtil.temporaryServerAddresses(3, "127.0.0.1").map(_.getPort)
   // See src/main/resources/application.conf for bootstrap settings which are used in docs so needs tested
   val config = ConfigFactory.parseString(s"""
       akka.remote.artery {

--- a/integration-test/local/src/test/scala/akka/management/LocalBootstrapTest.scala
+++ b/integration-test/local/src/test/scala/akka/management/LocalBootstrapTest.scala
@@ -17,9 +17,7 @@ import org.scalatest.time.{ Seconds, Span }
 import org.scalatest.{ Matchers, WordSpec }
 
 object LocalBootstrapTest {
-  val managementPorts = (0 until 3).map(
-    _ => SocketUtil.temporaryServerAddress(address = "localhost").getPort
-  )
+  val managementPorts = SocketUtil.temporaryServerAddresses(3, "127.0.0.1" ).map(_.getPort)
   // See src/main/resources/application.conf for bootstrap settings which are used in docs so needs tested
   val config = ConfigFactory.parseString(s"""
       akka.remote.artery {
@@ -30,21 +28,21 @@ object LocalBootstrapTest {
           port = 0
         }
       }
-      akka.management.http.hostname = "localhost"
+      akka.management.http.hostname = "127.0.0.1"
       akka.discovery {
         config.services = {
           local-cluster = {
             endpoints = [
               {
-                host = "localhost"
+                host = "127.0.0.1"
                 port = ${managementPorts(0)}
               },
               {
-                host = "localhost"
+                host = "127.0.0.1"
                 port = ${managementPorts(1)}
               },
               {
-                host = "localhost"
+                host = "127.0.0.1"
                 port = ${managementPorts(2)}
               }
             ]

